packages/babel-plugin-proposal-nullish-coalescing-operator/src/index.js

```diff
@@ -4,6 +4,7 @@ import { types as t, template } from "@babel/core";
 
 export default declare((api, { loose = false }) => {
   api.assertVersion(7);
+  const noDocumentAll = api.assumption("noDocumentAll") ?? loose;
```

Comment:
Are we switching all of these to assumption then lose?

Reply:
Yes, I'll update the other PRs

Reply:
The assumption, when unset, is `undefined` so any configured assumptions overwrites the loose config.